docs/machine-learning/how-to-guides/index.md

```diff
@@ -8,6 +8,33 @@ ms.date: 10/05/2018
 In the How to section of the ML.NET Guide you can find quick answers
 to common questions. In some cases, articles may be listed in multiple sections to make them easy to find.
```

Comment:
The articles listed in multiple sections sentence is confusing -- there are only two sections, the second has only one entry, and it doesn't duplicate any entries in the first section.

Reply:
Good feedback. I'll remove the headings and add to the list.